shared/chat/inbox/row/build-team/container.js

```diff
@@ -6,8 +6,7 @@ import type {TypedState, Dispatch} from '../../../../util/container'
 import BuildTeam from '.'
 
 const mapStateToProps = (state: TypedState) => ({
-  loaded: state.teams.loaded,
-  showBuildATeam: state.teams.teamnames.size === 0,
+  showBuildATeam: !state.chat2.metaMap.count(m => m.teamType !== 'adhoc'),
```

Comment:
`metaMap.some` seems more appropriate here. Also, could you store the `metaMap` in `stateProps` and move the check to `mergeProps`?